Source/Core/DolphinQt2/AboutDialog.cpp

```diff
@@ -54,8 +54,8 @@ AboutDialog::AboutDialog(QWidget* parent) : QDialog(parent)
   text_label->setOpenExternalLinks(true);
 
   QLabel* copyright =
-      new QLabel(tr("© 2003-%1 Dolphin Team. “GameCube” and “Wii” are"
-                    " trademarks of Nintendo. Dolphin is not affiliated with Nintendo in any way.")
+      new QLabel(tr("\u00A9 2003-%1 Dolphin Team. \u201cGameCubeu201d and \u201cWiiu201d are "
```

Comment:
Late, but were the escaping backslashes accidentally left out after "GameCube" and "Wii" for `\u201D`?